ui/packages/security-logout/src/main/resources/logout-response.html

```diff
@@ -26,9 +26,13 @@
 </head>
 <body class="full-width logout-response-body">
 <div class="full-width logout-response-container">
-    <div class="logout-msg">
-        <h2 id="extramessage"></h2>
-        <h4>You can <a id="landinglink" href="#">return to the landing page.</a></h4>
+    <div id="standard-msg" class="logout-msg" style="display:none;">
+        <h2>You have been logged out</h2>
+        <h4><a id="link" href="#">Sign in again</a></h4>
+    </div>
+    <div id="close-browser-msg" class="logout-msg" style="display:none;">
+        <h2>You have been logged out</h2>
+        <h4>Please close your browser to prevent reuse of cached credentials</h4>
```

Comment:
❓ I'm confused as to why we have both `index.html` and `logout-response.html`. It looks to my like they both display a similarly formatted logout page, but `index.html` is specifically for the error case. Is that how they are used in code, i.e. we return `logout-response.html` when logout succeeds and return `index.html` when logout fails?

Reply:
`index.html` is used by `main.js` it processes logout at the beginning by getting the logout URL and calling it. If at any point that fails it's responsible to terminate the process and display the logout error. 

`logout-response.html` is used by `logout-response.js` it is called at the end of logout and its task is to either display the successful logout message or a close your browser message